tests/hook_main.py

```diff
@@ -1118,7 +1122,10 @@ def event_type_post(transaction):
     :param transaction:
     :return:
     """
-    transaction['skip'] = True
+    with stash['app'].app_context():
```

Comment:
This is not required. To testing for POST, we don't have to create an object already. 
Rest looks good to me.